pkg/http/server.go

```diff
@@ -73,7 +73,6 @@ func (s *server) useTracing(next fasthttp.RequestHandler) fasthttp.RequestHandle
 }
 
 func (s *server) useMetrics(next fasthttp.RequestHandler) fasthttp.RequestHandler {
-	log.Infof("enabled metrics http middleware")
```

Comment:
Removed this log as it will always be printed even when disabled. We could add the log to Init in `InitMetrics` by passing the logger. Would that be preferred?

Reply:
Would want to make this consistent with the tracing log message too.